cli/connect/interactive.go

```diff
@@ -0,0 +1,244 @@
+package connect
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/formatter"
+)
+
+// setOutputFormatPrefix is a prefix for a set output format interactive option.
+const setOutputFormatPrefix = "\\set output "
+
+// setOutputFormatShortPrefix is a short command for a set output format interactive option.
+const setOutputFormatShortPrefix = "\\x"
+
+// setTableDialectPrefix is a short prefix for a set output table dialect interactive option.
+const setTableDialectPrefix = "\\set table_format "
+
+// setPseudoGraphicsEnable is an interactive option for pseudo graphics enabling of
+// table/ttalbe output formats.
+const setPseudoGraphicsEnable = "\\xG"
+
+// setPseudoGraphicsEnable is an interactive option for pseudo graphics disabling of
+// table/ttalbe output formats.
+const setPseudoGraphicsDisable = "\\xg"
+
+// setTableColWidthShortPrefix is a short prefix for table column width interactive option.
+const setTableColWidthShortPrefix = "\\xw "
+
+// setTableColWidthShortPrefix is a prefix for table column width interactive option.
+const setTableColWidthPrefix = "\\set table_column_width "
+
+// helpOptionHeandler prints help of tt connect interactive option.
+func helpOptionHeandler() {
+	var help string = `
+  To get help, see the Tarantool manual at https://tarantool.io/en/doc/
+  To start the interactive Tarantool tutorial, type 'tutorial()' here.
+
+  This help is expanded with additional backslash commands
+  because tt connect is using.
+
+  Available backslash commands:
+
+  \set language <language>   -- set language lua or sql
+  \set output <format>       -- set output format yaml, lua, table or ttable
+  \x[y,l,t,T]                -- set output format yaml, lua, table or ttable
+  \x                         -- switches output format cyclically
+  \x[g,G]                    -- disables/enables pseudographics for tables
+  \set table_column_width    -- max column width for table/ttable
+  \xw                        -- max column width for table/ttable
+  \table_format              -- tables format markdown, jira or default
+  \help                      -- show this screen
+  \quit                      -- quit interactive console
```

Comment:
It will be clearer:
```suggestion
  \set language <language>   -- set language lua or sql
  \set output <format>       -- set output format yaml, lua, table or ttable
  \x[y,l,t,T]                -- set output format yaml, lua, table or ttable
  \x                         -- switches output format cyclically
  \x[g,G]                    -- disables/enables pseudographics for tables
  \set table_column_width <width>    -- max column width for table/ttable
  \xw                        -- max column width for table/ttable
  \table_format <format>             -- tables format markdown, jira or default
  \help                      -- show this screen
  \quit                      -- quit interactive console
```

Reply:
Done